eli5/formatters/image.py

```diff
@@ -0,0 +1,326 @@
+# -*- coding: utf-8 -*-
+from typing import Union, Optional, Callable
+
+import numpy as np # type: ignore
+from PIL import Image # type: ignore
+import matplotlib.cm # type: ignore
+
+from eli5.base import Explanation
+
+
+# FIXME: in autodoc the interpolation and colormap defaults show
+# something like: 
+# interpolation=<Mock id='139885759460352'>, colormap=<Mock id='139885771653072'>
+# ID's instead of actual names.
```

Comment:
Maybe we can install matplotlib when building docs to fix this, and not mock it?

Reply:
Indeed installing `matplotlib` (and `PIL`) removed the `<Mock...` artefacts. 

Unfortunately autodoc actually evaluates the expressions so I get something like `interpolation=1, colormap=<matplotlib.colors.ListedColormap object>`. It is possible to override the function signature doc (https://stackoverflow.com/a/12087750/11555448), which is a solution that I would prefer.

Reply:
> It is possible to override the function signature doc

That looks great, thanks!

Reply:
https://github.com/TeamHG-Memex/eli5/pull/315/commits/88417feb979ac722d1fbbc9ffb8a5baf4fec365c